tsdef/types.d.ts

```diff
@@ -13,6 +14,7 @@ import { RemoteVideoTrackPublication } from './RemoteVideoTrackPublication';
 import { Track } from './Track';
 import { VideoTrack } from './VideoTrack';
 
+export { LocalTrackOptions } from './LocalTrackOptions';
```

Comment:
looks like this file does not use `LocalTrackOptions` anymore. Do we still need to import and export it from this file?

Reply:
@makarandp0 But `LocalTrackOptions` still need to be exported since it is a publicly available API.

Reply:
yes, and I see that it is already exported from index.d.ts (just like other not part of types.d.ts for example `LocalDataTrack`).  Looks like we need to export types form types.d.ts only if they were defined in that file.

Reply:
@makarandp0 we still have other types that use the `LocalTrackOptions` such as `MediaStreamTrackPublishOptions`.

Reply:
@PikaJoyce, you are right. I missed that reference. I guess importing `LocalTrackOptions` is needed. But does this file need to re-export it?

Reply:
@makarandp0 Ah, sorry! I see what you mean! I need to import into types but will move the export to `index.d.ts`.